packages/suite/src/components/firmware/Fingerprint.tsx

```diff
@@ -17,7 +17,11 @@ const Wrapper = styled.pre`
 const Fingerprint = ({ device }: { device: TrezorDevice }) => (
     <Wrapper>
         {/* device.firmwareRelease should be always defined here (this renders upon dispatching ButtonRequest_FirmwareCheck) */}
-        {getFormattedFingerprint(device.firmwareRelease!.release.fingerprint)}
+        {getFormattedFingerprint(
+            device.features!.major_version === 1 // support intermediary
```

Comment:
you could use `import { supportIntermediary } from '@suite-utils/device';` that you introduced and then the comment is no more needed...

Reply:
12a2a46